### PR TITLE
fix: improve accessibility of scripts

### LINKED ISSUE #1464: [BUG] improve accessibility with script supporting elements in unordered list
### Description

- As shown in the image
- improve accessibility by using `<li>` under `<ol>`

### Screenshots

image

### Any additional information?

_No response_

### What browser are you seeing the problem on?

_No response_

### Checklist

- [X] I have checked the existing issues
- [X] I have read the [Contributing Guidelines](https://github.com/rupali-codes/LinksHub/blob/main/CONTRIBUTING.md)
- [X] I am willing to work on this issue (optional)
- [ ] I am a GSSoC'23 contributor

### PR DESCRIPTION
**Pull request by CBID2**
## Fixes Issue
Closes #1464 

## Changes proposed

This PR adds ordered lists (`<ol>`) around the GitHub buttons and social media icons in the `SocialMediaIconsList` component, making the code more accessible for screen readers.

## Note to reviewers
Hey @Anmol-Baranwal. I've never really worked with Typescript before, so feel free to make the necessary adjustments.

### CODE REVIEW

File: components/SocialMedia/SocialMediaIconsList.tsx [hunk 1]
```diff
@@ -38,7 +42,9 @@ export const SocialMediaIconsList: FC<{
               <FaDiscord className="hover:text-theme-primary transition duration-300 ease-in-out" />
             </IconContext.Provider>
           </a>
+          </ol>
         </li>
```
Comment (Anmol-Baranwal) [inline components/SocialMedia/SocialMediaIconsList.tsx:46]:
```suggestion
          </li>
        </ol>
```

**Review by Anmol-Baranwal — CHANGES_REQUESTED** [review]
Check the order. This might be causing error in deployment.

**Comment by CBID2** [comment]
> Check the order. This might be causing error in deployment.

Just made the change @Anmol-Baranwal! :) It worked! :)

**Review by Anmol-Baranwal — APPROVED** [review]
Glad it worked :)

**Comment by CBID2** [comment]
Hey @rupali-codes. Can you merge this please?

**Review by rupali-codes — APPROVED** [review]

**Comment by rupali-codes** [comment]
> Hey @rupali-codes. Can you merge this please?

Sure, sorry for being late

**Pull request merged**